demo/api.md

```diff
@@ -738,6 +738,187 @@ Only for use with the `range` attribute. Sets the label for the second input.
 <!-- AURO-GENERATED-CONTENT:END -->
 </auro-accordion>
 
+### Day Slot
+
+Each day within the calendar has a slot that allows a user to add custom content. The user will add their slot content withtin the `auro-datepicker`, with the slot name needing to be in the format of `date_{month}_{day}_{fullYear}`. **Note**: When the month is a single digit, a zero will need to be added to the front of the month. For example, January would be `01` and not `1`.
+
+In order to style your slot content, we recommend using in-line styles on your component, as the day slot content is passed down multiple levels within the component. Another way to style the component is to target the `part="daySlot"` attribute that is added to the container around the slot content.
```

Comment:
```suggestion
For `<slot>` styling, it is recommended to use in-line styles. 

... WE NEED A HTML EXAMPLE HERE ....

Another option is to target the `daySlot` CSS Part. This will target the HTML node surrounding the slotted content. 

... WE NEED A CSS EXAMPLE HERE ....
```

Reply:
Unfortunately, an example using css parts cannot be added at this time. Due to the way the slot content is getting passed down to the auro-calendar-cell level on render, adding a css rule within the html for the specific part attribute does not target the attribute correctly, nor apply the style.

That being said, I completely agree with your requested change, I am just not sure how to correctly implement the example at this time.